packages/orbit-components/src/Tabs/components/Tab/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { convertHexToRgba } from "@kiwicom/orbit-design-tokens";
+
+import { useTabs, useTab } from "../../TabContext";
+import type { Props } from "./types";
+import { TYPE_OPTIONS } from "./consts";
+import defaultTheme from "../../../defaultTheme";
+
+const getColorBackground = ({ type, theme }) => {
+  if (type === TYPE_OPTIONS.MEDIUM) return theme.orbit.backgroundColorTabBundleMedium;
+  if (type === TYPE_OPTIONS.BASIC) return theme.orbit.backgroundColorTabBundleBasic;
+
+  return theme.orbit.backgroundColorTabBundleTop;
+};
+
+const getBackgroundType = (state?: "hover" | "active") => ({
```

Comment:
I don't think we need these functions. Can't we just use the correct values where they are needed? This is extra runtime cost

Reply:
It's then not readable and it has to be functions, otherwise, we would need to use nested ternary conditions.

Reply:
I doubt it's heavily affects runtime cost